packages/node_modules/@ciscospark/react-component-title-bar/src/index.js

```diff
@@ -4,20 +4,31 @@ import classNames from 'classnames';
 import styles from './styles.css';
 
 import Avatar from '@ciscospark/react-component-avatar';
+import Button from '@ciscospark/react-component-button';
 
+import {ICON_TYPE_WAFFLE} from '@ciscospark/react-component-icon';
 
-export default function TitleBar({name, image}) {
+export default function TitleBar({hasMenu, image, name, onMenuClick}) {
   return (
     <div className={classNames(`title-bar`, styles.titleBar)}>
       <div className={classNames(`avatar-container`, styles.avatarContainer)}>
-        <Avatar image={image} name={name} />
+        <Avatar image={image} name={name} size={`large`} />
       </div>
       <h1 className={classNames(`title`, styles.title)}>{name}</h1>
+      { hasMenu &&
+        <div className={classNames(`menu-container`, styles.menuContainer)}>
+          <div className={classNames(`button-container`, styles.buttonContainer)}>
+            <Button iconType={ICON_TYPE_WAFFLE} onClick={onMenuClick} />
+          </div>
+        </div>
+      }
     </div>
   );
 }
 
 TitleBar.propTypes = {
+  hasMenu: PropTypes.bool,
   image: PropTypes.string,
-  name: PropTypes.string.isRequired
+  name: PropTypes.string.isRequired,
+  onMenuClick: PropTypes.func
 };
```

Comment:
Or have a button that's attached a level up. I think I'm leaning this way because the way I think of components is isolating concerns as much as possible and this case feels like it should be 2 different components due to the fact that the menu button and the title bar are 2 separate features. Based on the zeplin comps for calling there is also a situation where the title bar isn't present but the menu button is available.